packages/oas-to-snippet/src/supportedLanguages.js

```diff
@@ -1,80 +1,212 @@
-const supportedLanguages = {
+module.exports = {
```

Comment:
I should note that by putting all these alternative targets into these configs that we're going to be exposing them in the `APIRequest` module with some work I'm doing to support custom code samples for manual APIs. Shouldn't be a problem, but FYI regardless.